Dockerfile

```diff
@@ -14,11 +14,11 @@ RUN wget https://dl.dropboxusercontent.com/u/4550074/mesos/mesos-dns+50fc45a9 -O
 ADD ./isolator /isolator/isolator/
 ADD ./m4 /isolator/m4/
 ADD ./bootstrap /isolator/
-ADD ./calico_isolator.py /isolator/
+ADD ./calico_isolator /isolator/
 ADD ./configure.ac /isolator/
 ADD ./Makefile.am /isolator/
 ADD ./requirements.txt /isolator/
-ADD http://downloads.mesosphere.io/demo/calico-node/v0.5.1/calico-node-v0.5.1.tar /isolator/
+COPY ./calico-node-v0.5.4.tar /isolator/
```

Comment:
This is up for debate, but the compose file relies on you having the calico-node-v0.5.4.tar in the project's root directory. I revived the makefile and `make images` to get that in place for you.
- If we remove this requirement, we have two options. 1 - don't cache calico-node. When an agent comes up, the first thing it will do is pull the image over the network. 
- Instead of placing the calico-node image in the root directory, we can make it accessible over the network (like we did with the demo). However, this means that every time docker-compose executes, it will pull the 1.5 gig file. Docker can't cache a network file - it has to download it first to see if the hash matches.

After debating it awhile, I went with the Makefile. But I am open to suggestions